src/commands/check-vulnerabilities.yml

```diff
@@ -92,7 +92,7 @@ steps:
         headers = {
          'Accept': 'application/json',
          'API-Key': os.path.expandvars("${<<parameters.api-key>>}"),
-         'Authorization': base64.b64encode(os.path.expandvars("${<<parameters.username>>}:${<<parameters.service-key>>}"))
+         'Authorization': base64.b64encode(os.path.expandvars("${CONTRAST_USERNAME}:${CONTRAST_SERVICE_KEY}").encode('UTF-8')).decode('ascii')
```

Comment:
@redcom Can you explain the change from `<<parameters.username>>` to `CONTRAST_USERNAME`

Reply:
I can switch back. My company wants to keep email private so that we do not expose it. I have added it to environment variables so I can easily test it. I guess it is better to switch back.